### ISSUE TITLE
NullPointerException on PushIOEngagementService

### ISSUE THREAD

**Issue by renato-almeida** [pushio/PushIOManager_Android#6]
Hi guys,

Just received a crash report from your app:
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.os.PowerManager$WakeLock.release()' on a null object reference
       at com.pushio.manager.PushIOEngagementService.onHandleIntent(PushIOEngagementService.java:60)
       at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:135)
       at android.os.HandlerThread.run(HandlerThread.java:61)
```

I can't really explain how this happened.

**Comment by neerhaj** [comment]
Hi Renato,

Kindly share the complete log if possible. That will help us determine the cause.

**Comment by renato-almeida** [comment]
I only have the crash report, it was caught with Crashalitycs.
I'm sorry

**Comment by neerhaj** [comment]
Hi Renato,

This issue has been fixed in release 2.13.4. See https://github.com/pushio/PushIOManager_Android/releases/tag/v2.13.4
